src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java

```diff
@@ -113,16 +113,19 @@ public static String getPagesPackage() {
      */
     public static int getTimeOut() {
         if (TIMEOUT == null) {
-            if (Props.get("page.load.timeout") == null || "".equals(Props.get("page.load.timeout"))) {
-                LOG.warn("Set timeout in your properties file, key 'page.load.timeout'. Now using default value "
-                        + defaultTimeout + " milliseconds.");
+            String pageLoadTimeoutProp = Props.get("page.load.timeout");
+            if (pageLoadTimeoutProp == null || "".equals(pageLoadTimeoutProp)) {
```

Comment:
change it to Props.get("page.load.timeout").isEmpty()
In all code

Reply:
And we have method 
    public static String get(String prop, String defaultValue) 
You can provide default value directly to a method.
Also, you can implement an override like 
    public static String get(String prop, String defaultValue, String logOnDefault)
You should pass can logOnDefault string into logger like 
   get("prop", "true", "Using default value %s for this property");
And inside the method put it into:
LOG.warn(logOnDefault, defaultValue);